client/container_logs_test.go

```diff
@@ -116,17 +127,15 @@ func TestContainerLogs(t *testing.T) {
 			}),
 		}
 		body, err := client.ContainerLogs(context.Background(), "container_id", logCase.options)
-		if err != nil {
-			t.Fatal(err)
+		if logCase.expectedError != "" {
+			require.EqualError(t, err, logCase.expectedError)
```

Comment:
Should probably be `assert.EqualError()` in this case so it runs the other cases even if one case fails.

Reply:
yes, I can update that; I was trying to keep the existing behaviour, but agree updating would make sense.